Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9-alpine as production
+FROM python:3.9.6-alpine as production
```

Comment:
@Uxio0 this is an interesting one... maybe an issue with Dependabot? 👀

Reply:
Looks like, yes, but I think that the check is not really easy with all the aliases

Reply:
Maybe we can dismiss this change then 👍